lock-api-objects/src/main/java/com/palantir/lock/watch/LockWatchReferences.java

```diff
@@ -34,22 +34,33 @@ private LockWatchReferences() {
         // no
     }
 
+    // Matches any lock descriptor relating to the given table.
     public static LockWatchReference entireTable(String qualifiedTableRef) {
         return ImmutableEntireTable.of(qualifiedTableRef);
     }
 
+    // Matches any row or cell lock descriptor relating to the given table,
+    // as long as the row name starts with the prefix.
+    // Currently, this reference type is unsupported.
     public static LockWatchReference rowPrefix(String qualifiedTableRef, byte[] rowPrefix) {
         return ImmutableRowPrefix.of(qualifiedTableRef, rowPrefix);
     }
 
+    // Matches a range of rows in a given table, e.g. from "cat" to "dog".
+    // Currently, this reference type is unsupported.
     public static LockWatchReference rowRange(String qualifiedTableRef, byte[] startInclusive, byte[] endExclusive) {
         return ImmutableRowRange.of(qualifiedTableRef, startInclusive, endExclusive);
     }
 
+    // Matches row or cell descriptors with row name exactly matching the row bytes.
+    // Note that this does not match rows starting with the row bytes.
     public static LockWatchReference exactRow(String qualifiedTableRef, byte[] row) {
         return ImmutableExactRow.of(qualifiedTableRef, row);
     }
 
+    // Matches precisely one AtlasCellLockDescriptor. Cell-level conflict handling (RETRY_ON_WRITE_WRITE_CELL or
+    // SERIALIZABLE_CELL) is required for this to match anything.
```

Comment:
I'm not 100% certain on this, but I believe that if you don't have this kind of conflict checking, you'll take out RowLockDescriptors and not CellLockDescriptors, and therefore exact_cell wouldn't match (not that it's supported anyway...)